MAINTAINERS

```diff
@@ -7,7 +7,7 @@ In alphabetical order:
 Kingdon Barrett, Weaveworks <kingdon@weave.works> (github: @kingdonb, slack: Kingdon B)
 Daniel Holbach, Weaveworks <daniel@weave.works> (github: @dholbach, slack: dholbach)
 Stefan Prodan, Weaveworks <stefan@weave.works> (github: @stefanprodan, slack: stefanprodan)
-Scott Rigby, Weaveworks <scott@weave.works> (github: @scottrigby, slack: scottrigby)
+Scott Rigby, Weaveworks <scott@r6by.com> (github: @scottrigby, slack: scottrigby)
```

Comment:
Maybe this as well?

```suggestion
Scott Rigby, Independent <scott@r6by.com> (github: @scottrigby, slack: scottrigby)
```

:people_hugging:

Reply:
race condition 😸